src/TLuaInterpreter.cpp

```diff
@@ -88,6 +88,8 @@ TLuaInterpreter::TLuaInterpreter( Host * pH, int id )
 : mpHost( pH )
 , mHostID( id )
 , purgeTimer(this)
+, mpGatekeeperThread()
+, mpLuaSessionThread()
```

Comment:
Although this does seem to be used in `(void) TLuaInterpreter::startLuaSessionInterpreter()` that function is never called so it is all just cruft...!